core/modules/server/server.js

```diff
@@ -322,8 +326,16 @@ Server.prototype.listen = function(port,host,prefix) {
 		port = process.env[port] || 8080;
 	}
 	// Warn if required plugins are missing
-	if(!this.wiki.getTiddler("$:/plugins/tiddlywiki/tiddlyweb") || !this.wiki.getTiddler("$:/plugins/tiddlywiki/filesystem")) {
-		$tw.utils.warning("Warning: Plugins required for client-server operation (\"tiddlywiki/filesystem\" and \"tiddlywiki/tiddlyweb\") are missing from tiddlywiki.info file");
+	var missing = [];
+	for (let index = 0; index < this.variables["required-plugins"].length; index++) {
+		const r = this.variables["required-plugins"][index];
```

Comment:
We don't use `let` or `const` atm and please give `r` a real name. ... For me the next line looked like a "broken" regexp ...

Reply:
code style. `for` should look like

```
for (var index=0; index<this.variables["required-plugins"].length; index++) {
```

eg: no spaces between `index=0;`

Reply:
Thanks @pmario. I have been bouncing back and forth between different code styles, and this is really helpful!